workspace_host/interface.js

```diff
@@ -54,6 +54,9 @@ if ('config' in argv) {
 }
 config.loadConfig(configFilename);
 
+const workspaceUid = config.workspaceJobsDirectoryOwnerUid;
+const workspaceGid = config.workspaceJobsDirectoryOwnerGid;
```

Comment:
This will cause this bug again: https://github.com/PrairieLearn/PrairieLearn/pull/2887